artifactory.py

```diff
@@ -1676,6 +1679,7 @@ def deploy(
         parameters={},
         explode_archive=None,
         explode_archive_atomic=None,
+        check_sum=None,
```

Comment:
also, need to add docstrings explaining usage and parameters

Reply:
Docstring is added